public/locales/fr/translation.json

```diff
@@ -291,6 +292,13 @@
           "title": "Abandonner les modifications ?",
           "body": "Vous allez perdre toutes les modifications que vous avez faites sur les paramètres de scénario."
         }
+      },
+      "about": {
+        "title": "Webapp générique",
```

Comment:
Can we reuse the application title `Exemple d'Application Web Azure` to stay consistent ?